3p/integration.js

```diff
@@ -39,6 +39,7 @@ import {
 } from './3p';
 import {urls} from '../src/config';
 import {endsWith} from '../src/string';
+import {parseJson} from '../src/json';
```

Comment:
If people complain we can fix it :)

We also have `parseUrl` and `parseQueryString`, so there is reasons to do either.